src/FastRandom.cpp

```diff
@@ -8,8 +8,11 @@
 
 #include <random>
 
-#if defined (__GNUC__)
-	#define ATTRIBUTE_TLS static __thread
+#if defined (ANDROID)
+	// Tycho: I'm sorry, I'm sorry, I'm so, so, sorry, but please blame the NDK developers for lack of, thread_local
+	#define ATTRIBUTE_TLS static
```

Comment:
Apparently only GCC emulates thread local storage and not Clang (yet), but GCC crashes when compiling Lua.

Also, imagining your expression when you read the downgrading of TLS to just the plain static, and then saying, "really?", makes me laugh :P